applications/nrf5340_audio/tools/buildprog/pcft_sign.py

```diff
@@ -278,19 +282,46 @@ def sign(orig_pcft_hex, build_dir):
     shutil.copy(f'{build_dir}/zephyr/net_core_app_update.bin'.format(),
                 f'{BIN_DIR}/{final_file_prefix}net_core_app_update_{fw_info_firmware_version}.bin')
 
+def find_hex_name(options):
+
+    in_path = Path(options.input_folder).resolve()
+
+    # Sort hex files
+    hex_files_found = sorted(in_path.glob('*.hex'))
+    pattern_found = []
+    if options.min_b0n:
+        for file in hex_files_found:
+            if re.match(ORGG_PCFT_MIN_PATTERN, file.name):
```

Comment:
Maybe do an inline if based on ORIG_PCFT_(...)?

Reply:
Replace with List Comprehension style.
`pattern_found = [
    file for file in hex_files_found if re.match(ORIG_PCFT_MIN_PATTERN if options.min_b0n else ORIG_PCFT_PATTERN, file.name) is not None
]
`